bdd100k/common/utils.py

```diff
@@ -84,7 +87,7 @@ def list_files(inputs: str) -> List[List[str]]:
     return files_list
 
 
-def read(inputs: str) -> List[List[DictAny]]:
+def read(inputs: str) -> List[List[Frame]]:
     """Read annotations from file/files."""
```

Comment:
We can use `scalabel.label.io` here

Reply:
The JSON files we have in hand for BDD don't strictly follow the format definitions. But you can see some specific settings in this function